styles.css

```diff
@@ -0,0 +1,58 @@
+
+.mobile-header {
+  background-color: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-right: 48px;
+}
+
+.mobile-header a {
+  display: flex;
+  justify-content: center;
+  text-decoration: none;
+  margin: 15px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-size: 15px;
+  color: #fff;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  color: #ff6b00;
+  padding: 120px 24px;
+  background-color: #1c1a19;
+  background-image: url('./img1.png');
```

Comment:
- [ ] This image is not present in your repository, hence it's not loaded. Kindly add it